src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -1105,6 +1109,40 @@ class spell_pal_seal_of_righteousness : public AuraScript
     }
 };
 
+// 42463 - Seal of Vengeance
+// 53739 - Seal of Corruption
+class spell_pal_seal_of_vengeance : public SpellScript
+{
+    PrepareSpellScript(spell_pal_seal_of_vengeance);
+
+    void HandleScriptEffect(SpellEffIndex /*effIndex*/)
+    {
+        Unit* target = GetExplTargetUnit();
+        uint32 spellId = GetSpell()->GetSpellInfo()->Id;
+        uint32 auraId = (spellId == SPELL_PALADIN_SEAL_OF_VENGEANCE_EFFECT)
+            ? SPELL_PALADIN_HOLY_VENGEANCE
+            : SPELL_PALADIN_BLOOD_CORRUPTION;
```

Comment:
Validate the spells

Reply:
Unfamiliar with spell scripts, not sure if I did it right.

Reply:
Yup. Rule of thumb is that if a spell is used within a script it has to be validated